scheduler/src/api/jobs/get_job.rs

```diff
@@ -30,9 +31,17 @@ pub struct GetJobResponse {
     job: JobWithSubJobsWithData,
 }
 
+#[derive(Serialize, ToSchema)]
+pub struct DownloadSpeed {
+    sub_job_id: Uuid,
+    is_partial: bool,
```

Comment:
What's the reason for putting this here? I'd guess that it's enough that it's in the sub_job details and if we duplicate it here we risk some inconsistencies.

Reply:
I wanted to create a short `summary`, actual job response is large. 
That said, do we actually need to know if speed result is a partial?

Reply:
>  That said, do we actually need to know if speed result is a partial?

Nope, number of workers that executed this sub_job should be enough (and actually preferable, just bool `is_partial` won't be enough to run regression).